blogs/2021/10/11/webview-ui-toolkit.md

```diff
@@ -6,17 +6,18 @@ MetaDescription: Announcing the public preview of the Webview UI Toolkit for Vis
 Date: 2021-10-11
 Author: David Dossett
 ---
+
 # Webview UI Toolkit for Visual Studio Code
 
 October 11, 2021 by David Dossett, [@david_dossett](https://twitter.com/david_dossett) and Hawk Ticehurst [@hawkticehurst](https://twitter.com/hawkticehurst)
 
-We're so excited to announce the public preview of the [Webview UI Toolkit for Visual Studio Code](https://github.com/microsoft/vscode-webview-ui-toolkit). With this, extensions developers can quickly and easily create webview-based extensions in Visual Studio Code that look, feel, and act like the editor itself.
+We're so excited to announce the public preview of the [Webview UI Toolkit for Visual Studio Code](https://github.com/microsoft/vscode-webview-ui-toolkit). With this, extensions developers can quickly and easily create [webview-based extensions](https://code.visualstudio.com/api/extension-guides/webview) in Visual Studio Code that look, feel, and act like the editor itself.
 
 ![Webview UI Toolkit for Visual Studio Code: gallery view of components](webview-ui-toolkit-artwork.png)
 
 ## What is the Webview UI Toolkit?
 
-At its core, the toolkit is a library of components that developers can use to build user interfaces inside extension webviews.
+At its core, the toolkit is a library of components that developers can use to build user interfaces inside extension webviews. It's important to understand that webviews should only be used if you absolutely need them. Take a look at the [webview guidelines](https://code.visualstudio.com/api/references/extension-guidelines#webviews) to learn more about when it is appropriate to use a webview.
```

Comment:
Rather than "it's important to understand", can you soften this to something like "Note that this doesn't change our recommendation around avoiding the use of webviews in extensions unless you absolutely need them"?

Reply:
Updated